debian/control

```diff
@@ -4,7 +4,8 @@ Priority: optional
 Maintainer: Leonardo Held <leonardo.held@toradex.com>
 Build-Depends: debhelper (>= 9),
                lsb-release,
-               wget
+               wget,
+	       ca-certificates
```

Comment:
Change the initial tab for spaces:

```suggestion
               ca-certificates
```

If you fell like it, put on alphabetical order after the debhelper stuff:

```diff
diff --git a/debian/control b/debian/control
index 02566f4..0d7812c 100644
--- a/debian/control
+++ b/debian/control
@@ -3,9 +3,9 @@ Section: devel
 Priority: optional
 Maintainer: Leonardo Held <leonardo.held@toradex.com>
 Build-Depends: debhelper (>= 9),
+               ca-certificates
                lsb-release,
                wget,
-              ca-certificates
 Standards-Version: 4.5.1
 Homepage: https://www.nxp.com/lgfiles/NMG/MAD/YOCTO/libgpuperfcnt-6.4.3.p4.2-aarch64-wayland.bin
 
```